### PR TITLE
types: type optimization

### PR DESCRIPTION
**Pull request by kiner-tang**
- fix CI Broken: https://github.com/ant-design/ant-design/actions/runs/3899011193/jobs/6658303742

### CODE REVIEW

File: src/Notifications.tsx [hunk 1]
```diff
@@ -66,10 +66,10 @@ const Notifications = React.forwardRef<NotificationsRef, NotificationsProps>((pr
         // Replace if exist
         const index = clone.findIndex((item) => item.key === config.key);
         if (index >= 0) {
-          config.times = (config.times || 0) + 1;
+          config.__times = (config.__times || 0) + 1;
```
Comment (zombieJ) [inline src/Notifications.tsx:69]: 不用下划线

Comment (kiner-tang) [reply src/Notifications.tsx]: done

File: src/useNotification.tsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ export interface NotificationAPI {
 
 interface OpenTask {
   type: 'open';
-  config: OpenConfig;
+  config: OpenConfig & { times?: number };
```
Comment (zombieJ) [inline src/useNotification.tsx:36]: 这个应该不用对外暴露的

Comment (kiner-tang) [reply src/useNotification.tsx]: done

**Pull request merged**